### PR TITLE
meteor-git: init at 0.22.0 (and add self to maintainer-list.nix)

### LINKED ISSUE #332795: Package request: meteor
**Project description**

It's a conventional commits git helper that has a gorgeous TUI

**Metadata**

* homepage URL: https://github.com/stefanlogue/meteor
* source URL: https://github.com/stefanlogue/meteor
* license: mit
* platforms: unix, linux, darwin, (basicaly anything with go)

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by nebunebu**
## Description of changes

https://github.com/stefanlogue/meteor
closes: https://github.com/NixOS/nixpkgs/issues/332795

## Things done

add `nixpkgs/pkgs/by-name/me/meteor-git/package.nix`

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/me/meteor-git/package.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    owner = "stefanlogue";
+    repo = "meteor";
+    rev = "v${version}";
+    hash = "sha256-aY/gOKvcKtOnL4FI2SM339LU4HoWYCq0W9mK2GyMqso=";
+  };
+
+  vendorHash = "sha256-jKd/eJwp5SZvTrP3RN7xT7ibAB0PQondGR3RT+HQXIo=";
+
+  meta = {
+    description = "A highly configurable CLI tool for writing conventional commits ";
```
Comment (afh) [inline pkgs/by-name/me/meteor-git/package.nix:21]: According to the [recommendations for meta attributes](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes) the description should not start with an article and avoid subjective language.
```suggestion
    description = "Configurable CLI tool for writing conventional commits ";
```

**Review by afh — CHANGES_REQUESTED** [review]
Thanks for this, @nebunebu.

Can you rework the commits so that there are two commits: one for adding you to the maintainers list and one for the meteor-git package and the commit adding you as a maintainer comes before the meteor-git commit, please.

**Comment by afh** [comment]
Result of `nixpkgs-review pr 335282` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>meteor-git</li>
  </ul>
</details>

**Comment by nebunebu** [comment]
@afh I don't have any darwin machines. I have split up the commit that adds myself to the maintaier-list and adds the package into two commits

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -14403,6 +14403,12 @@
     githubId = 2287221;
     name = "Andreas Zweili";
   };
+  nebunebu = {
+    email = "neb.nebuchadnezzar@gmail.com";
+    github = "nebunebu";
+    githubId = "87451010";
```
Comment (afh) [inline maintainers/maintainer-list.nix:14409]: `githubId` should be numeric:
```suggestion
    githubId = 87451010;
```

**Review by afh — CHANGES_REQUESTED** [review]
There's no need to have darwin machines, folks that do, like me, will test PRs 🙂 

Please also change the commit message to `maintainers: add nebunebu`

**Comment by afh** [comment]
Result of `nixpkgs-review pr 335282` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>meteor-git</li>
  </ul>
</details>

**Comment by afh** [comment]
Thanks for making the changes, @nebunebu, much appreciated. Any particular reason you dislike or oppose the [changes suggested to `meta.description`](https://github.com/NixOS/nixpkgs/pull/335282/files#r1720734199)?

**Comment by nebunebu** [comment]
@afh I just overlooked it while making other changes.

**Comment by tomberek** [comment]
Result of `nixpkgs-review pr 335282` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>meteor-git</li>
  </ul>
</details>

**Comment by afh** [comment]
Thanks for addressing all the suggestions, @nebunebu, much appreciated and welcome to nixpkgs 🙂

**Pull request merged**